### ISSUE TITLE
Dockerfile build fails

### ISSUE THREAD

**Issue by ioiste** [w4/bin#18]
Hi! 
Thanks for having _bin_ opensource!
I'm trying to build the Dockerfile, but it seems like it can't import something. Would you be able to take a look, please?
```
   Compiling yaml-rust v0.4.4
   Compiling owning_ref v0.4.1
error[E0432]: unresolved import `crate::arch::x86_64::__m64`
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/api/into_bits/arch_specific.rs:51:15
```

The full errors after this are available here https://bin.gy/hickudylle.
Thank you!

**Comment by 31KM** [comment]
Happens to me as well with no docker build:
```
Some errors have detailed explanations: E0432, E0554.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `packed_simd`
```

**Comment by KoxSosen** [comment]
Hm.. 
```
   Compiling devise_core v0.3.0 (https://github.com/SergioBenitez/Devise.git?rev=1e42a2691#1e42a269)
Some errors have detailed explanations: E0432, E0557.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `packed_simd` due to 8 previous errors
warning: build failed, waiting for other jobs to finish...
Some errors have detailed explanations: E0432, E0557.
For more information about an error, try `rustc --explain E0432`.
error: build failed
```

**Comment by w4** [comment]
This is fixed in the move to actix-web in 2.0.0, packed_simd is no longer apart of our dependency graph
